routes/warehousesRoutes.js

```diff
@@ -1,31 +1,10 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const knex = require('../knexfile');
+const knex = require("../knexfile");
+const { createNewWarehouse } = require("../controllers/warehousesController");
+const { deleteWarehouse } = require("../controllers/warehousesController");
 
-// DELETE /api/warehouses/:id
-router.delete('/:id', async (req, res) => {
-  const warehouseId = req.params.id;
+router.post("/warehouse", createNewWarehouse);
+router.delete("/api/warehouses/:id", deleteWarehouse);
```

Comment:
There is an issue with how the routes are defined in the warehousesRoutes.js file. Currently, these two routes:
```
router.post("/warehouse", createNewWarehouse);
router.delete("/api/warehouses/:id", deleteWarehouse); 
```
they will concatenate/join  with the base route defined in server.js, resulting in the following incorrect paths:
```
POST /api/warehouses/warehouse
DELETE /api/warehouses/api/warehouses/:id
```

The correct routes should be:
```
POST /api/warehouses
DELETE /api/warehouses/:id
```
To fix this, since you’ve already set /api/warehouses as the base in the server.js file, you just need to modify the routes in warehousesRoutes.js like this:

```
router.post("/", createNewWarehouse);
router.delete("/:id", deleteWarehouse);
```

This ensures the correct full route paths are generated, with /api/warehouses coming from the server.js file and concatenating correctly.

Reply:
that makes a lot of sense thank you for explaining in details! I've updated the routes now

Reply:
nice work

Reply:
thanks  and nice work